client/src/app/domain/models/poll/poll-constants.ts

```diff
@@ -131,7 +131,21 @@ export const PollTypeVerbose = {
 
 export type PollTypeVerboseKey = keyof typeof PollTypeVerbose;
 
-export const PollPropertyVerbose = {
+export type PollPropertyVerboseKey =
+    | `onehundred_percent_base`
+    | `type`
+    | `pollmethod`
+    | `state`
+    | `groups`
+    | `votes_amount`
+    | `global_yes`
+    | `global_no`
+    | `global_abstain`
+    | `max_votes_amount`
+    | `min_votes_amount`
+    | `max_votes_per_option`;
+
+export const PollPropertyVerbose: Record<PollPropertyVerboseKey, string> = {
```

Comment:
You don't need to specify a type here. 
```suggestion
export const PollPropertyVerbose = {
```

Reply:
Working on it. The reason we don't want a type definition here, is because it would be overkill.